sdk/browser/test/integration/otp/index.test.ts

```diff
@@ -4,26 +4,17 @@ import '../env'
 
 import { expect } from 'chai'
 import { __dangerous } from '@affinidi/wallet-core-sdk'
-import { getOtp, getOptionsForEnvironment } from '../../helpers'
-
+import { MessageParameters } from '@affinidi/wallet-core-sdk/dist/dto'
 import { AffinityWallet } from '../../../src/AffinityWallet'
 
-const signedCredentials = require('../../factory/signedCredentials')
-const { openAttestationDocument } = require('../../factory/openAttestationDocument')
-
-const DELAY = 1000
-// prettier-ignore
-const wait = (ms: any) => new global.Promise(resolve => setTimeout(resolve, ms))
-
-const generateEmail = () => {
-  const TIMESTAMP = Date.now().toString(16).toUpperCase()
+import { getOptionsForEnvironment } from '../../helpers'
+import { TestmailInbox } from '../../../test/helpers/TestmailInbox'
+import { openAttestationDocument } from '../../factory/openAttestationDocument'
 
-  return `test.user-${TIMESTAMP}@gdwk.in`
-}
+const signedCredentials = require('../../factory/signedCredentials')
```

Comment:
You can also use import there :)

Reply:
It fails and just didn't want to mess more than I already did :D
Says "it's not a module" or something like that.

Reply:
Ah, of course, because `signedCredentials` is indeed not a modern module for some reason: https://github.com/affinityproject/affinidi-core-sdk/blob/master/sdk/browser/test/factory/signedCredentials.ts#L67

OK, let's leave it like that in this PR